bin/Sherpa/cards/production/13TeV/higgs/Run.dat_13TeV_gamgam_forint_interference

```diff
@@ -0,0 +1,46 @@
+ (run){
+ EVENTS 100000;
+ EVENT_GENERATION_MODE P;
+ ME_SIGNAL_GENERATOR Amegic Higgs;
+ RANDOM_SEED XX; XX:=0; WH:=0.00407;
+ ANALYSIS Rivet; ANALYSIS_OUTPUT results/Analysis_XX_WH_QSF; ANALYSIS_WRITEOUT_INTERVAL 0.1;
```

Comment:
Please remove this line,
For the sherpacks I don't think Rivet analysis output is necessary